skbio/io/_exception.py

```diff
@@ -84,6 +84,11 @@ class QSeqFormatError(FileFormatError):
     pass
 
 
+class StockholmFormatError(FileFormatError):
+    """Raised when a ``stockholm`` formatted file cannot be parsed."""
+    pass
```

Comment:
Also recommend leaving as-is since the other exceptions in this module are defined that way.